autowebcompat/network.py

```diff
@@ -3,6 +3,9 @@
 from keras.models import Model
 from keras.optimizers import SGD, Adam, Nadam, RMSprop
 
+SUPPORTED_NETWORKS = ['inception', 'vgglike', 'vgg16']
+SUPPORTED_OPTIMIZERS = ['sgd', 'adam', 'nadam', 'rms']
```

Comment:
We should merge this and `allOptimizers` from `compile` into a single variable, but we can do it in a follow-up.